Debian Bug #777590
Currently is_distro_package() is returning False. Need to investigate why it is broken on Debian
Index: apport/apport/ui.py
===================================================================
--- apport.orig/apport/ui.py	2015-01-14 14:49:38.000000000 +0530
+++ apport/apport/ui.py	2015-02-10 16:36:46.247797028 +0530
@@ -124,8 +124,11 @@
                     report['UnreportableReason'] = _('This package does not seem to be installed correctly')
                 elif not apport.packaging.is_distro_package(report['Package'].split()[0]):
                     # TRANS: %s is the name of the operating system
-                    report['UnreportableReason'] = _(
-                        'This is not an official %s package. Please remove any third party package and try again.') % report['DistroRelease'].split()[0]
+                    #report['UnreportableReason'] = _(
+                    #    'This is not an official %s package. Please remove any third party package and try again.') % report['DistroRelease'].split()[0]
+                    pass
+                    # See Debian Bug #777590
+
         except ValueError:
             # this happens if we are collecting information on an uninstalled
             # package
